api/core/v1alpha1/digest.proto

```diff
@@ -8,7 +8,7 @@ package core.v1alpha1;
 // A digest of content or globally unique content identifier
 message Digest {
   DigestType type = 1;
-  bytes value = 2;
+  string value = 2;
```

Comment:
please add a comment here how the value is constructed and an example (e.g. we expect a hex string here for the digest value)